app/services/chat_publisher.rb

```diff
@@ -130,6 +130,17 @@ def self.publish_channel_edit(chat_channel)
     )
   end
 
+  def self.publish_channel_status(chat_channel)
```

Comment:
I think later on we can handle some sort of dynamic reaction to this. Perhaps for now a reload of the page is enough?

Reply:
> dynamic reaction

Do you mean something like booting people from the channel or disabling the chat box?

Reply:
Yeah basically changing the UI in response to the status changing. The server-side guardian checks will make sure that no new messages get through, just not sure how we want the user to see this change. Maybe will ask in the topic.